app/javascript/components/pages/SongEdit.test.js

```diff
@@ -20,8 +21,13 @@ describe("When SongEdit renders", () => {
     expect(songEditHeading).toEqual("Update or Delete Song Information");
   });
 
+  it("displays a song form", () => {
+    const songForm = edit.find("Form");
+    expect(songForm.length).toEqual(1);
+  });
+
   xit("displays song title", () => {
```

Comment:
I know we commented this out, but I encourage you to keep at it as you'll have a much easier time developing with good tests.